cli/filplus.go

```diff
@@ -375,21 +375,23 @@ var filplusListClaimsCmd = &cli.Command{
 			tablewriter.Col("TermMin"),
 			tablewriter.Col("TermMax"),
 			tablewriter.Col("TermStart"),
+			tablewriter.Col("Expiration"),
 			tablewriter.Col("Sector"),
 		)
 
 		for claimId, claim := range claimsMap {
 			if ts.Height() > claim.TermMax || !cctx.IsSet("expired") {
```

Comment:
I think it's this line that's the problem - `--expired` needs to check `ts.Height` against the expiration height of the claim; I don't think we need an expiration column, at least that would be anther breaking change and they can calculate it anyway